test/Qowaiv.DomainModel.UnitTests/Must_specs.cs

```diff
@@ -0,0 +1,75 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using Qowaiv.DomainModel;
+using Qowaiv.Validation.Abstractions;
+using System;
+
+namespace Must_specs
+{
+    public class Requires
+    {
+        [Test]
+        public void Subject_not_to_be_null()
+        {
+            Func<object> create = () => new Must<TestModel>(null);
+            create.Should().Throw<ArgumentNullException>();
+        }
+    }
+
+    public class Guards_when
+    {
+        [Test]
+        public void Must_Be_condition_is_met()
+        {
+            var model = new TestModel();
+            new Must<TestModel>(model).Be(condition: true, ValidationMessage.Error("This is wrong"))
+                .Should().BeValid().WithoutMessages()
+                .Value.Should().BeSameAs(model);
+        }
+
+        [Test]
+        public void Must_NotBe_condition_is_not_met()
```

Comment:
This double negation is very confusing

Reply:
Any suggestion? `NotBe` is - in this case - defacto just a term, although I see your point.

Reply:
Maybe, Must_NotBe_model_not_equal_to_condition? I find it hard to think of an easy to read title for this.

Reply:
if you put in `Math.Squared(4) != 2` the same happens. It is not model dependent per se.